pkg/identityserver/config.go

```diff
@@ -77,6 +77,9 @@ type Config struct {
 		SMTP         smtp.Config          `name:"smtp"`
 		Templates    emailTemplatesConfig `name:"templates"`
 	} `name:"email"`
+	Gateways struct {
+		EncryptionKeyID string `name:"encryption-key-id" description:"ID of the key used to encrypt gateway secrets at rest"`
```

Comment:
New config needs to be documented in `reference/config` and mentioned in `CHANGELOG`.

Reply:
https://github.com/TheThingsIndustries/lorawan-stack-docs/pull/25